pkg/chunk/cassandra/storage_client.go

```diff
@@ -34,6 +34,8 @@ type Config struct {
 	SSL                      bool                `yaml:"SSL"`
 	HostVerification         bool                `yaml:"host_verification"`
 	CAPath                   string              `yaml:"CA_path"`
+	CertPath                 string              `yaml:"cert_path"`
+	KeyPath                  string              `yaml:"key_path"`
```

Comment:
We're trying to uniform config. Could you rename the YAML config options and their CLI flags respectively to:

- `tls_cert_path` / `tls-cert-path`
- `tls_key_path` / `tls-key-path`